streaming/src/test/scala/org/apache/spark/streaming/InputStreamsSuite.scala

```diff
@@ -27,7 +27,8 @@ import scala.collection.JavaConverters._
 import scala.collection.mutable
 
 import com.google.common.io.Files
-import org.apache.hadoop.fs.Path
+import org.apache.commons.io.IOUtils
```

Comment:
I'll look @ that; I think I went with IOU as it was on the CP and I'd never had bad experiences of it. Guava, well...